README.md

```diff
@@ -73,3 +64,27 @@ cellfinder -s signal_images -b background_images -o output_dir --metadata metada
 ```
 
 Full documentation can be found [here](https://brainglobe.info/documentation/cellfinder/index.html).
+
+
+## Developers documentation
+
+This repository also includes workflow scripts that are benchmarked to support code development. We regularly benchmark the time the workflows take to complete to ensure performance is stable as the code changes.
```

Comment:
```suggestion
This repository also includes workflow scripts that are benchmarked to support code development.
These benchmarks are run regularly to ensure performance is stable, as the tools are developed and extended.
```

Also a handy tip for markdown files; start every sentence on a newline (if you're not in a list) - it means that hunting through the diff for changes and edits is a little bit faster :slightly_smiling_face:

Reply:
I was wondering if there was a pre-commit hook for this but didn't find any already made

Reply:
There isn't one sadly, but I have my VSCode setup with a markdown linter which helps standardise some things.
https://marketplace.visualstudio.com/items?itemName=yzhang.markdown-all-in-one